gsoc.yml

```diff
@@ -1,3 +1,34 @@
+gsoc2020:
+  theand9:
+    rss_fed: "rss-url" # TODO (add blog with gsoc tag)
+    project: 'stingray'
+  pkj-m:
+    rss_feed: "https://pkj-m.github.io/blog/feed.gsoc.xml"
+    project: 'radis'
+  siddharthlal25:
+    rss_feed: "https://siddharthlal25.github.io/blog/feed.gsoc.xml" # TODO (add blog with gsoc tag)
+    project: 'JuliaAstro'
+  meuge:
+    rss_feed: "https://blog.poliastro.space/feeds/meuge.rss.xml"
+    project: 'poliastro'
+  JeS24:
+    rss_feed: "https://dev.to/feed/jes24"
```

Comment:
@JeS24 does dev.to allows getting the feed only for particular tags? could you update this with a feed to a gsoc tag?

Reply:
@dpshelio dev.to, like Medium, does not allow per-user tag-based feeds, i.e. `/feed/[user]/tag/[tag]` doesn't work. Something like https://dev.to/feed/tag/gsoc seems like the only option for tag-based feeds, but then, it has a multitude of posts, tagged `gsoc`. Should I update `gsoc.yml` with this? 

Also, I will be using this dev.to blog only for GSoC 2020-related posts, if that helps!

Reply:
No, that would bring everyone with such tag. We will need to update the code in the `grab.py` to accommodate for `dev.to` It shouldn't be too difficult. Though you may be using it only for GSoC, you may want to use it also to blog about other stuff.. and I hope more people in the future uses dev.to instead of medium, anyway, so I am happy to try to generalise it now.

Reply:
There's no way on `dev.to` to get the tags (thepracticaldev/dev.to#1128). So we will grab everything you post - at least during GSoC.